chapel.html.markdown

```diff
@@ -396,9 +429,9 @@ for value in realArray {
 writeln(rSum, "\n", realArray);
 
 // Associative arrays (dictionaries) can be created using associative domains.
-var dictDomain: domain(string) = { "one", "two" };
-var dict: [dictDomain] int = ["one" => 1, "two" => 2];
-dict["three"] = 3; // Adds 'three' to 'dictDomain' implicitly
```

Comment:
The point of this is to show that  keys can be added to an associated array.
Has this been changed in new Chapel versions?